uds/uds_config_tool/odx/pos_response.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict, List
+
+from uds.uds_config_tool import DecodeFunctions
+from uds.uds_config_tool.odx.param import Param
+
+
+class PosResponse():
```

Comment:
```suggestion
class PositiveResponse:
```

Reply:
I used PosResponse so it is consistent with the XML Element in ODX. Do you think its better to change it anyway?

Reply:
Fixed declaration, kept name